test/integration/execute.spec.ts

```diff
@@ -609,6 +615,42 @@ describe('.execute()', () => {
 		);
 		expect(threadResult.error).toBe(false);
 
+		// Before enqueing up a create event we need to create a relationship to the random-slug type created above
```

Comment:
Same as other related comment.

Reply:
This test explicitly creates a new `card` with a random slug instead of using an existing type like `card`.
I'm not sure why it does it this way but I prefer for this PR to not update this test logic.

Reply:
I think we're doing that as a way to test creating events on threads. Normally that would be done using the `message`/`whisper` type, but the worker doesn't know about those so its creating a temporary replacement. In that case, then I think you're right to be adding in a new temporary relationship to match.